arlo-client/src/components/AuditForms/SelectBallotsToAudit.tsx

```diff
@@ -151,35 +152,45 @@ const SelectBallotsToAudit: React.FC<IProps> = ({
         const body = {
           size, // until multiple contests are supported
         }
-        await api(`/election/${electionId}/audit/sample-size`, {
+        const response: IErrorResponse = await api(
```

Comment:
Casting this as an `IErrorResponse` is confusing. If you follow my other suggestions, I think you could just make `IErrorResponse` have a non-optional `errors` field and use it as a distinct type from valid responses.